src/ServiceControlInstaller.Engine/Instances/ServiceControlInstance.cs

```diff
@@ -159,6 +157,15 @@ public string AclUrl
             }
         }
 
+        public string AclMaintenanceUrl
+        {
+            get
+            {
+                var baseUrl = $"http://{HostName}:{MaintenancePort}/";
```

Comment:
Should we take the virtual directory into account?

Reply:
Nope